firefox-ios/Providers/Profile.swift

```diff
@@ -598,6 +598,8 @@ open class BrowserProfile: Profile {
                     case .tabReceived(_, let tabData):
                         let url = tabData.entries.last?.url ?? ""
                         return URL(string: url, invalidCharacters: false)
+                    case .tabsClosed:
```

Comment:
If you're open to a suggestion:

```suggestion
                    default:
```

That way, any commands that we add in the future won't cause build errors. It's always OK for a device to ignore any command it doesn't understand.

Reply:
I wasn't sure if we should be explicit so we have more awareness on new cases, so thank you for confirming that its okay to ignore future commands!

Reply:
Of course! It would be so nice if we could use `@unknown default` and get the best of both worlds here, but we [can't build Rust Components with library evolution support yet](https://github.com/mozilla/application-services/blob/main/docs/adr/0003-swift-packaging.md#problems-with-the-current-setup) (the `BUILD_LIBRARY_FOR_DISTRIBUTION` flag), so I think Swift will treat `IncomingDeviceCommand` as implicitly frozen and complain that the `@unknown default` is redundant.